### ISSUE TITLE
Have not resize train image to 256*256 in ImageNet

### ISSUE THREAD

**Issue by xiaodongww** [pytorch/examples#274]
https://github.com/pytorch/examples/blob/master/imagenet/main.py#L119-L126

In the code:
`    train_dataset = datasets.ImageFolder(
        traindir,
        transforms.Compose([
            transforms.RandomResizedCrop(224),
            transforms.RandomHorizontalFlip(),
            transforms.ToTensor(),
            normalize,
        ]))`

I think there need a resize operation, like followings:
`train_dataset = datasets.ImageFolder(
        traindir,
        transforms.Compose([
            transforms.Resize(256),
            transforms.RandomResizedCrop(224),
            transforms.RandomHorizontalFlip(),
            transforms.ToTensor(),
            normalize,
        ]))`
Is that right?

Thanks~

**Comment by SsnL** [comment]
I'm confused. Do you want to resize to 256 or not? Do you want to still do random 224 crops?

**Comment by xiaodongww** [comment]
Sorry about that,  I just mixstake `torchvision.RandomResizedCrop` for `torch.RandomCrop`. My bad.

Nice work you have done!

Best regrads!
